tests/rest/admin/test_room.py

```diff
@@ -1430,6 +1430,54 @@ def test_join_private_room_if_owner(self):
         self.assertEquals(200, int(channel.result["code"]), msg=channel.result["body"])
         self.assertEqual(private_room_id, channel.json_body["joined_rooms"][0])
 
+    def test_context(self):
```

Comment:
Can you add a test where this returns a 401 if we try and call if with a normal user access token please